service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/NoticeOfPlacementOrderUploadedEventHandler.java

```diff
@@ -46,15 +49,15 @@ public void notifyParties(NoticeOfPlacementOrderUploadedEvent noticeOfPlacementE
         notificationService.sendEmail(
             NOTICE_OF_PLACEMENT_ORDER_UPLOADED_TEMPLATE, emails, notifyData, caseData.getId().toString());
 
-        issuedOrderAdminNotificationHandler.notifyAdmin(caseData,
-            noticeOfPlacementEvent.getDocumentContents(), NOTICE_OF_PLACEMENT_ORDER);
+        issuedOrderAdminNotificationHandler.notifyAdmin(
+            caseData, orderDocument.getBinaryUrl(), NOTICE_OF_PLACEMENT_ORDER);
```

Comment:
minor, for consistence with e.g changes below it would be better to pass  orderDocument. It is type safe and better prepare for future changes. If e.g we would like do display name of doc in emai, then when passing entire DocReference, no changes to interface are needed

Reply:
I agree, thanks, will pass whole document